### PR TITLE
Add codegen to integration test

### PR DESCRIPTION
**Pull request by moritzkiefer-da**
Currently these tests are somewhat slow due to using a fresh Maven database each time. I will move the creation of the Maven cache to a separate Maven rule so that it gets better caching in a separate PR.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-assistant/integration-tests/BUILD.bazel [hunk 1]
```diff
@@ -4,26 +4,43 @@ load("//bazel_tools:haskell.bzl", "da_haskell_test")
...
+        "//daml-lf/archive:daml_lf_archive_java.jar",
+        "//daml-lf/archive:daml_lf_archive_java_pom.xml",
+        "//language-support/java/bindings:bindings-java",
+        "//language-support/java/bindings:bindings-java_pom.xml",
+        "//language-support/java/bindings-rxjava",
+        "//language-support/java/bindings-rxjava:bindings-rxjava_pom.xml",
+        "//language-support/java/codegen:shaded_binary.jar",
+        "//language-support/java/codegen:shaded_binary_pom.xml",
+        "//ledger-api/rs-grpc-bridge:librs-grpc-bridge.jar",
+        "//ledger-api/rs-grpc-bridge:rs-grpc-bridge_pom.xml",
```
Comment (gerolf-da) [inline daml-assistant/integration-tests/BUILD.bazel]: I guess this is not trying to be an exhaustive list or artefacts, is it?

Comment (moritzkiefer-da) [reply daml-assistant/integration-tests/BUILD.bazel]: This is only a list of dependencies that are required by the quickstart example. I verified that it is sufficient by changing the VERSION file to some bogus version that is not yet published to bintray.

Comment (gerolf-da) [reply daml-assistant/integration-tests/BUILD.bazel]: okay, thank you.

**Pull request merged**